cset-workflow/flow.cylc

```diff
@@ -119,7 +119,7 @@ URL = https://metoffice.github.io/CSET
 
     [[install_local_cset]]
     # Install CSET from source.
-    execution time limit = PT5M
+    execution time limit = PT10M
```

Comment:
Out of curiosity, do you experience this timing out often? In theory it should take less than 30 seconds...

Reply:
This was due to the `.tox` hidden folder being copied, which contained a bunch of large conda environments.